src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/SplitButtonTests/SplitButtonPage.xaml.cs

```diff
@@ -0,0 +1,199 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Windows.Input;
+using Uno.UI.Samples.Controls;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+using SplitButton = Microsoft.UI.Xaml.Controls.SplitButton;
+//TODO: using SplitButtonTestApi = Microsoft.UI.Private.Controls.SplitButtonTestApi;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5246693)